modules/flowable-cmmn-rest/src/main/java/org/flowable/cmmn/rest/service/api/runtime/caze/CaseInstanceIdentityLinkResource.java

```diff
@@ -72,9 +78,15 @@ public void deleteIdentityLink(@ApiParam(name = "caseInstanceId") @PathVariable(
             @ApiParam(name = "type") @PathVariable("type") String type,
             HttpServletResponse response) {
 
-        CaseInstance caseInstance = getCaseInstanceFromRequest(caseInstanceId);
+        CaseInstance caseInstance = runtimeService.createCaseInstanceQuery().caseInstanceId(caseInstanceId).singleResult();
+        if (caseInstance == null) {
+            throw new FlowableObjectNotFoundException("Could not find a case instance with id '" + caseInstanceId + "'.");
+        }
 
         validateIdentityLinkArguments(identityId, type);
+        if (restApiInterceptor != null) {
+            restApiInterceptor.deleteCaseInstanceIdentityLink(caseInstance, identityId, type);
```

Comment:
Lets do this a bit more specific like when fetching a specific link. So just before we delete the link we can do `deleteCaseInstanceIdentityLink(caseInstance, link)`, where `link` comes from the `getIdentityLink` call.

Reply:
done